test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -2214,21 +2222,18 @@ describe('the rubicon adapter', function () {
           expect(imp.video.w).to.equal(640);
           expect(imp.video.h).to.equal(480);
           expect(imp.video.pos).to.equal(1);
-          expect(imp.video.context).to.equal('instream');
           expect(imp.video.minduration).to.equal(15);
           expect(imp.video.maxduration).to.equal(30);
           expect(imp.video.startdelay).to.equal(0);
           expect(imp.video.skip).to.equal(1);
           expect(imp.video.skipafter).to.equal(15);
-          expect(imp.ext.rubicon.video.playerWidth).to.equal(640);
-          expect(imp.ext.rubicon.video.playerHeight).to.equal(480);
-          expect(imp.ext.rubicon.video.size_id).to.equal(201);
-          expect(imp.ext.rubicon.video.language).to.equal('en');
+          expect(imp.ext.prebid.bidder.rubicon.video.playerWidth).to.equal(640);
+          expect(imp.ext.prebid.bidder.rubicon.video.playerHeight).to.equal(480);
+          expect(imp.ext.prebid.bidder.rubicon.video.language).to.equal('en');
+
           // Also want it to be in post.site.content.language
           expect(post.site.content.language).to.equal('en');
-          expect(imp.ext.rubicon.video.skip).to.equal(1);
-          expect(imp.ext.rubicon.video.skipafter).to.equal(15);
```

Comment:
There were some defaults or someting for these rubicon video params, it probably is ok we removed them but we should be sure.

Reply:
From bret's comments in https://magnite.atlassian.net/browse/HB-15147 : 

> imp.ext.rubicon.video.skip - I think this line of code is deeply unnecessary. “data.imp[0].video.skip = bidRequest.params.video.skip;” Forget it.

> imp.ext.rubicon.video.skipafter - I think this line of code is deeply unnecessary. “data.imp[0].video.skipafter = bidRequest.params.video.skipdelay;;” Forget it.

Reply:
👍